ext/standard/dir.c

```diff
@@ -160,6 +160,8 @@ PHP_FUNCTION(dir)
 static php_stream* php_dir_get_directory_stream_from_user_arg(php_stream *dir_stream)
 {
 	if (dir_stream == NULL) {
+		php_error_docref(NULL, E_DEPRECATED,
+			"Passing null is deprecated, instead the last opened directory stream should be provided");
```

Comment:
I think "the last opened" could be replaced by "a" here.

Reply:
The objective of the message is to indicate how to get rid of the deprecation _without_ altering behaviour.
Replacing "the last opened" by "a" is unhelpful here.